spryker/README.md

```diff
@@ -2,7 +2,7 @@
 
 In a Dockerfile for PHP and NGINX:
 ```Dockerfile
-ARG PHP_VERSION=7.3 # 7.1 and 7.2 are also available.
+ARG PHP_IMAGE_VERSION=7.3 # 7.1 and 7.2 are also available.
 FROM quay.io/continuouspipe/spryker-php${PHP_VERSION}-nginx:stable
```

Comment:
The `ARG` above only applies to this `FROM`, so needs to be used as PHP_IMAGE_VERSION or ARG adjusted to PHP_VERSION

Reply:
:man_facepalming: Sorry, missed that bit too, yep, I renamed so it was consistent with the build arg naming, updated.